modules/fun.js

```diff
@@ -0,0 +1,450 @@
+// @ts-check
+const Augur = require(`augurbot-ts`),
+  Discord = require(`discord.js`),
+  u = require(`../utils/utils`),
+  axios = require('axios'),
+  mineSweeperEmojis = { 0:'0⃣', 1:'1⃣', 2:'2⃣', 3:'3⃣', 4:'4⃣', 5:'5⃣', 6:'6⃣', 7:'7⃣', 8:'8⃣', 9:'9⃣', 10:'🔟', 'bomb':'💣' };
+/**
+ * function color
+ * @param {Discord.ChatInputCommandInteraction} int a /fun color interaction
+ */
+async function slashFunColor(int) {
+  let colorCode = int.options.getString(`color`);
+  if (!colorCode) {
+    colorCode = `#${Math.floor(Math.random() * 16777216).toString(16).padStart(6, '0')}`;// generate random hex color
+  }
+  try {
+    const Jimp = require(`jimp`);
+
+    let colorCSS;
+    if (colorCode.startsWith('0x')) {
+      // In the case that we have a string in 0xABCDEF format
+      colorCSS = `#${colorCode.substring(2)}`;
+    } else {colorCSS = colorCode;}
+    if (![`#000000`, `black`, `#000000FF`].includes(colorCSS)) {
+      colorCSS = Jimp.cssColorToHex(colorCSS);
+    }
```

Comment:
Reduce the number of brackets
```suggestion
    let colorCSS = colorCode;
    // In the case that we have a string in 0xABCDEF format
    if (colorCode.startsWith('0x')) colorCSS = `#${colorCode.substring(2)}`;
    // For some reason, black results in a value of 255.
    if (![`#000000`, `black`, `#000000FF`].includes(colorCSS)) colorCSS = Jimp.cssColorToHex(colorCSS);
```